lib/asciidoctor-diagram/plantuml/extension.rb

```diff
@@ -10,12 +10,10 @@ module PlantUml
 
       private
 
-      JARS = ['plantuml.jar'].map do |jar|
-        File.expand_path File.join('../..', jar), File.dirname(__FILE__)
-      end
-      Java.classpath.concat JARS
-
       def plantuml(parent, source, tag, mime_type)
+
+        plantuml_jar = which_jar(parent, source, "plantuml", :raise_on_error => false)
```

Comment:
Shouldn't we raise an error if plantuml can not be found? Otherwise you'll get pretty obscure errors (NoClassDefFoundError probably) from the Java wrapper code.